snooty.toml

```diff
@@ -13,4 +13,7 @@ node-driver-long = "MongoDB Node.js Driver"
 node-driver-short = "Node.js Driver"
 
 java-version = "4.9"
+java-version-next = "4.10" # Not needed when Java v4.10 is released
+java-version-full = "{+java-version-next+}.0" # Update to use {java-version} when Java v4.10.x is released
```

Comment:
Suggestion:

If the PR is meant to be merged and published when the Java v4.10 drivers are released, it would probably be fine to update the java-version source constant to "4.10" instead of creating a "-next" and referencing it in "-full". If it is meant to be merged prior, then feel free to ignore.

Reply:
I thought this was going to be published before Java v4.10, but now it's looking like we're putting on hold until Kotlin GA / Java v4.10 release -- Ashni to confirm. If that's the case, I'll strip the '(Alpha)` label and update these source constants, then wait to merge until the GA release.